src/hpc/autoscale/job/job.py

```diff
@@ -35,6 +39,22 @@ def _check_type(**kwargs: Any) -> None:
     )
 
 
+def _initialize_job_config() -> None:
+    config_path = os.getenv("AUTOSCALE_JOB_CONFIG", "/opt/cycle/pbspro/allocate.conf")
+    cwd = os.getcwd()
+    logging.info("config_path %s", config_path)
```

Comment:
This looks like debug logging that should be removed, no?

Reply:
yep, done